main.rb

```diff
@@ -1,7 +1,7 @@
 $LOAD_PATH.push File.expand_path('../lib', __FILE__)
 require 'billing_reporter'
 
-if ARGV.length < 5
+if ARGV.length != 5
   puts "Usage: ruby main.rb destination_directory year month total_monthly_auths.json sp.yml"
```

Comment:
If we were starting from scratch, I'd recommend using something like `optparse` might make it easier to specify arguments, then you could do like `--year 2020 --month 10` which is more readable and then harder to confuse which argument is which

Reply:
That is true and i use it all the time but it's far more useful when the arguments are optional

Reply:
If this is a show stopper for you I'll code it up....

Reply:
Nope! Not a show stopper! That's what I meant when I said "if we were starting from scratch"